plugins/collections/README.md

```diff
@@ -53,3 +53,23 @@ Example payload:
     "pipelineIds": [12, 35, 47, 89]
 }
 ```
+
+#### Update a collection
+You can update the name, description, or pipelineIds of a collection.
+
+`PUT /collections/{id}`
+
+**Arguments**
+
+* `name` - An optional new name for the collection. Names must be unique.
+* `description` - An optional new description for the collection.
+* `pipelineIds` - An optional new array of ids of pipelines to be added to the collection.
```

Comment:
I think it's fine to have that in this route, that's how `PUT` is supposed to be used. My concern is more with the comments/docs about `pipelines to be added to the collection`, that makes it sound like the ones in the request will be added rather than replacing the current pipelines entirely.